docs/source/tutorials/datalab/datalab_quickstart.ipynb

```diff
@@ -700,12 +698,12 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "This portion of the info shows overall label quality summaries of all examples annotated as a particular class (e.g. the `Label Issues` column is the estimated number of examples labeled as this class that should actually have a different label).\n",
-    "To learn more about this, see the documentation for the `cleanlab.dataset.rank_classes_by_label_quality` method.\n",
+    "To learn more about this, see the documentation for the [cleanlab.dataset.rank_classes_by_label_quality](../../cleanlab/dataset.html#cleanlab.dataset.rank_classes_by_label_quality)\n",
+    "method.\n",
```

Comment:
this'll work across both stable/ and master/ branch versions of docs?

Reply:
It's a relative link so I think it should?